### DISCUSSION TITLE
Change config from velero client/agent

### DISCUSSION THREAD

**Discussion by 2020testuser** [vmware-tanzu/velero#3418] [Community support Q&A]
Hi,
I'm not sure that this is actually a bug.If the feature is not available, please consider this as an "Enhancement Request".  Looking for some insight.

1. I have velero server installed in my Kubernetes cluster.
I have around 10 project teams using Kubernetes and each have their own namespace.

Is it possible to overrider the s3 bucket name when I run velero client from my machine (locally)? I want each project team to take their own backup of their namespaces and store in their own s3 bucket. 

If the above is possible, could you please send me the command/syntax?

Thanks in advance!

**Comment by zubron** [comment]
Hi @2020testuser,

In order to backup to different buckets you will need to configure multiple [backup storage locations](https://velero.io/docs/v1.5/locations/) which each use a different bucket. Please see this answer on another discussion thread for details on this: https://github.com/vmware-tanzu/velero/discussions/3331#discussioncomment-320865

**Comment by 2020testuser** [comment]
@zubron,
Thanks! 
Let me make sure I got this. If I have the same s3 URL, I can use different buckets in the s3. However, all buckets need to use the same s3 credential? Is that correct?
If I have different credentials and use the same s3 with different buckets, then, there is a limitation. 
Could you please confirm?

Thanks!

**Reply by zubron** [reply]
@2020testuser Yes, that is correct. The limitation on using the same credential for each bucket will be fixed in Velero 1.6.

You can also use the same bucket but have a different "prefix" within each bucket for project team, like a directory within a bucket for each team. Please see our docs on the different configuration options for AWS S3 storage: https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/main/backupstoragelocation.md
